src/ploomber/telemetry/telemetry.py

```diff
@@ -129,16 +174,26 @@ def check_stats_enabled():
             return f"Error: Can't read config file {e}"
 
 
+def check_first_time_usage():
+    config_path = os.path.join(check_dir_file_exist(CONF_DIR), 'config.yaml')
+    uid_path = os.path.join(check_dir_file_exist(CONF_DIR), 'uid.yaml')
+    return not os.path.exists(uid_path) and os.path.exists(config_path)
```

Comment:
shouldn't this be `not os.path.exists(uid_path) and not os.path.exists(config_path)`?

Reply:
No since the call is being performed after the check for a stats config file (check the calling function)